<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DayFinder</class>
 <widget class="QMainWindow" name="DayFinder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DayFinder</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>110</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimumDateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>1752</year>
      <month>9</month>
      <day>12</day>
     </datetime>
    </property>
    <property name="displayFormat">
     <string>d/M/yyyy</string>
    </property>
    <property name="date">
     <date>
      <year>2000</year>
      <month>12</month>
      <day>31</day>
     </date>
    </property>
   </widget>
   <widget class="QLabel" name="dateLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter the date</string>
    </property>
   </widget>
   <widget class="QLabel" name="outputLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>411</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string>:empty:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>87</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate!</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
